erpnext/stock/dashboard/item_dashboard_list.html

```
@@ -50,14 +50,14 @@
 					data-stock-uom="{{ d.stock_uom }}"
-					data-item="{{ escape(d.item_code) }}">{{ __("Move") }}</a>
+					data-item="{{ d.item_code }}">{{ __("Move") }}</a>
 				{% endif %}
 				<button style="margin-left: 7px;" class="btn btn-default btn-xs btn-add"
 					data-disable_quick_entry="{{ d.disable_quick_entry }}"
 					data-warehouse="{{ d.warehouse }}"
 					data-actual_qty="{{ d.actual_qty }}"
 					data-stock-uom="{{ d.stock_uom }}"
-					data-item="{{ escape(d.item_code) }}"
+					data-item="{{ d.item_code }}"
 					data-rate="{{ d.valuation_rate }}">{{ __("Add") }}</a>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)